catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -81,6 +81,10 @@
 	public static final File TEMP_LOOK_FILE =
 			new File(DEFAULT_ROOT_DIRECTORY, "temporary_look_file" + DEFAULT_IMAGE_EXTENSION);
 
+	// Recent Bricks Directory
+	public static final String RECENT_BRICKS_DIRECTORY = "recent_bricks";
```

Comment:
This field is never used in your source code. This ticket does not really have to do something with the Backpack - so it would be nice if you actually used this directory instead of interweaving things with the Backpack, which is very confusing.